app/classifier.py

```diff
@@ -1,18 +1,56 @@
+from operator import itemgetter
+import logging
+import string
+import time
+import os
+
+import numpy as np
+import tensorflow as tf
+
 from app import embedder
+from app import model_fetcher
+
+
+def fetchVocab(vocab: str):
+    with open(vocab, 'r') as f:
+        return f.readlines()
 
 
 class Classifier:
 
-    def __init__(self):
-        self.embedder = embedder.Embedder()
+    def __init__(self, bert, classifier, vocab: str, prefetch=False):
+        self.logger = logging.getLogger('app.logger')
+        if prefetch:
+            f = model_fetcher.Fetcher()
+            filenames = f.fetchAll()
+            for f in filenames:
+                self.logger.info(f)
+        self.vocab = fetchVocab(vocab)
```

Comment:
shouldn't there be a tensorflow function for reading vocabs?

Reply:
Added a TODO to come up with a better serialization/unserialization strategy when we start needing to train models ourselves from Uwazi thesauri.